CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 0.2.10 (Common, Node.js, Web)
+
+### Breaking changes
+
```

Comment:
Is it a breaking change not for all `readonly` users, only for `readonly=1`? https://clickhouse.com/docs/en/operations/settings/constraints-on-settings